Azure/azurehpc-health-checks
# Addition of NvBandwidth and Installation refactoring

- Added Nvbandwidth tool to measure gpu BW
  - Modify install script (including directories)
  - Create custom test
  - modify applicable conf files
 
- Install refactor
  - Add applicable dependencies. Moved dependency install to earlier in the script.
  - Installation of external projects now done in functions
  - Added ability to change install location by passing path to install script.
  - Added ability to change cuda location by argument
  - Modified custom tests to point to given install directory
  - Modified customTests to work with all install changes
  -
- Update Readme
  - Fix expected value table
  - Update on install instructions
  - Update dependencies and give more info on dependencies installed
